workflows/scRNAseq/fastq-to-matrix-10x/scrna-seq-fastq-to-matrix-10x-v3.ga

```diff
@@ -31,7 +31,7 @@
     "format-version": "0.1",
     "license": "MIT",
     "release": "0.6",
-    "name": "scRNA-seq_preprocessing_10X_v3_Bundle",
+    "name": "Single-Cell RNA-seq Preprocessing: 10X Genomics v3 to Seurat-Compatible Format",
```

Comment:
@lldelisle isn't this also compatible with Scanpy? In that case, it is good to add Scanpy too because it is the most used toolsuite on Galaxy.

Reply:
The format (mtx, genes.tsv, barcodes.tsv) is compatible with Scanpy but the nested collection is probably not the easiest way to import them into scanpy. In fact, what should be done is to have the last step optional (re-arrange STAR solo output) so it is more importable in scanpy. The way it is now is perfect for download and post processing locally. But currently I have not time for this... Sorry.

Reply:
We could also just promote the scanpy compatible dataset/collection(?) to an output, then as a user you just pick what you need / the tool form would only let you pick the compatible input?

Reply:
I think the default output collections (compatible with Read10x) are just hidden. So it would be easy to do.